src/services/groupConfigApi.ts

```diff
@@ -19,6 +19,19 @@ export const getGroupConfigs = (requestConfig: AxiosRequestConfig = {}) => {
   return pncClient.getHttpClient().get<GroupConfigPage>('/group-configs', requestConfig);
 };
 
+/**
+ * Gets all Group Configs with no assigned Product Version.
+ *
+ * @param requestConfig - Axios based request config
+ */
+export const getUnassignedGroupConfigs = (requestConfig: AxiosRequestConfig = {}) => {
+  // TODO: extend request config with the helper function
```

Comment:
This should be rebased on top of https://github.com/project-ncl/pnc-web-ui-react/pull/328 which is already merged